### ISSUE TITLE
Conditional questions do not appear due to display:none styling not being removed.

### ISSUE THREAD

**Issue by GunsRoseSS** [umbraco/Umbraco.Forms.Issues#1117]
Hi all,

After updating my Umbraco Forms to 10.5.2 I've noticed that conditional questions do not appear anymore, despite the conditional logic being hit and applied correctly.

Reproduction
------------

### Bug summary

I believe this is due to an update to the frontend. Previously a `style="display: none;"` was added to hide the field, and this style was removed when the question needed to be visible. In 10.5.2 this is done by adding the `umbraco-forms-hidden` class, which applies the same styling.
The problem is that the default templates have not yet been updated in the docs for version 10.5.2. This results in a `style="display: none;"` being applied instead of the `umbraco-forms-hidden` class, making the question permanently invisible.

By updating the Form.cshtml template this bug will be resolved.
For example, on line 37,
`<fieldset class="umbraco-forms-fieldset" id="@fs.Id" @{ if (hideFieldSetWhenRendering) { <text> style="display: none" </text>  } }>`
should be changed to
`<fieldset class="umbraco-forms-fieldset @{ if (hideFieldSetWhenRendering) { <text> umbraco-forms-hidden </text>  }" id="@fs.Id"  }>`
or something like that.

Seems like a quick fix, I've just changed my custom theme file for now.

### Specifics

Umbraco version: 10.6.1
Forms version: 10.5.2

### Steps to reproduce

1. Create a new or use an existing Umbraco site with Umbraco Forms. Use the latest templates provided by the [docs](https://docs.umbraco.com/umbraco-forms/developer/themes).
2. Create a form with a conditional question. Make sure that the conditional flow is set up correctly.
3. Render the form on a page and test the condition.

### Expected result

The question should appear once the condition for it has been hit.

### Actual result

The question does not appear. There are not related errors.

**Comment by AndyButland** [comment]
Hi @GunsRoseSS - with Forms 10 we ship the default templates as part of the `Umbraco.Forms.StaticAssets` Razor class library, and they have been updated in there.  I've just checked under the release tag and see what you have suggested on line 37 of `Form.cshtml`.

```
<fieldset class="umbraco-forms-fieldset@{ if (hideFieldSetWhenRendering) { <text> umbraco-forms-hidden</text>  } }" id="@fs.Id">
```

If you have a custom theme, you will need to update these.  We provide a link to the latest version of theme from the Umbraco documentation [here](https://docs.umbraco.com/umbraco-forms/developer/themes#obtaining-the-default-theme-files), and via [this short link](https://umbra.co/umbraco-forms-default-theme).  I've just double-checked and it contains the updates.
